### DISCUSSION TITLE
Problem with installing Elementary OS

### DISCUSSION THREAD

**Discussion by clesioteixeira** [elementary/triage#507] [Q&A]
I'm having problems installing Elementary OS 7.1. When I try to install elementary OS, choosing the default installer option, the system stops at the terminal login screen, free for me to enter username and password and it stays there. When I choose the safe graphics option, the installer starts, I can install the system, but when I restart the installed and even updated system, the system also stops at the login screen (terminal) and doesn't go, it stays frozen, with the keyboard frozen. This happened every time I tried to install the system.

**Comment by vjr** [comment]
do you have nvidia GPU by any chance? anyway can you try providing the "safe graphics" option which is the kernel boot option "nomodeset" during the bootup after you install too? then you can search online for solutions to installing GPU drivers assuming that was the issue.

BTW during installation there is an option to install extra components like GPU / WiFi drivers etc I dont remember whether it is available for 7.1 though.

**Reply by clesioteixeira** [reply]
Yes, I have Nvidia. I'll test, I'll install Elementary OS again and add "nomodeset" to grub.

**Comment by blondejefe** [comment]
This has been an issue for nearly 2 months. Please update if nomodeset fixed your problem, it did not here.
